contributing_to_docs/contributing.adoc

```diff
@@ -140,21 +145,25 @@ documentation after 6 hours.
 ====
 All OpenShift content development occurs on the `master`, or *working* branch.
```

Comment:
This is technically the case from when 4.0 launches and for current 3.11. I don't know whether to keep it here till then, if it is wrong. I vote to keep it as most folks who are contributing know the difference.

Reply:
Right. I also don't want to assume that we're going to remember to do a good scrub on these files before we GA.